chat-commands.js

```diff
@@ -877,9 +878,9 @@ exports.commands = {
 		}
 	},
 	privateroomhelp: [
-		`/secretroom - Makes a room secret. Secret rooms are visible to & and up. Requires: & ~`,
-		`/hiddenroom [on/off] - Makes a room hidden. Hidden rooms are visible to % and up, and inherit global ranks. Requires: \u2606 & ~`,
-		`/publicroom - Makes a room public. Requires: \u2606 & ~`,
+		`/secretroom - Makes a room secret. Secret rooms are visible to & and up. Requires: & ~.
+		/hiddenroom [on/off] - Makes a room hidden. Hidden rooms are visible to % and up, and inherit global ranks. Requires: \u2606 & ~.
```

Comment:
by the way, you dont need periods at the end of all of the help command entries

Reply:
However, don't delete them if they were there before you added them; only delete ones you added.